pkg/ibmvpc/examples/out/explain_out/vpeToIksNodeNoProtocolConn_all_vpcs_explain_debug.txt

```diff
@@ -0,0 +1,30 @@
+Connectivity explanation for protocol: ICMP between 192.168.40.5 (iks-clusterid:1[192.168.40.5]) and 192.168.0.4 (iks-node[192.168.0.4]) within ky-test-vpc
+===========================================================================================================================================================
+
+There is no connection "protocol: ICMP" between iks-clusterid:1[192.168.40.5] and iks-node[192.168.0.4];
+ connection blocked by egress
+Egress: security group kube-r006-d7cfb31a-1d4b-40c8-83df-ce2e6f8f2e57 blocks connection; network ACL ky-test-edge-acl allows connection
+Ingress: network ACL ky-test-private-2-others-acl allows connection; security group kube-clusterid:1 allows connection; security group ky-test-default-sg allows connection
+
+Path:
+	iks-clusterid:1[192.168.40.5] -> |
+
+
+Details:
+~~~~~~~~
+Egress:
+security group kube-r006-d7cfb31a-1d4b-40c8-83df-ce2e6f8f2e57 blocks connection since there are no relevant allow rules
```

Comment:
How is this SG `kube-r006-d7cfb31a-1d4b-40c8-83df-ce2e6f8f2e57` associated with `kube-clusterid:1` ? 
Shouldn't it be `kube-clusterid:1` ? 
The convention was that we identify the associated SG by the prefix "kube-" and then the cluster ID .
from the name of the iks node, it seems the cluster id was changed to `clusterid:1` ?
so I would expect the SG name to also be changed?
@olasaadi99  can you check?

Reply:
> How is this SG `kube-r006-d7cfb31a-1d4b-40c8-83df-ce2e6f8f2e57` associated with `kube-clusterid:1` ? Shouldn't it be `kube-clusterid:1` ? The convention was that we identify the associated SG by the prefix "kube-" and then the cluster ID . from the name of the iks node, it seems the cluster id was changed to `clusterid:1` ? so I would expect the SG name to also be changed? @olasaadi99 can you check?

I changed the pool Id with `clusterid:1`, `r006-d7cfb31a-1d4b-40c8-83df-ce2e6f8f2e57` was not relevant to my changes, commit `[e5bc95e](vscode-webview://073p1f54fp1ap9p4m3jh3qfp3entst322a8eic0j94brt6pj7eic/index.html?id=99634360-c035-479f-bcf6-74ceab34c8ea&origin=ca35c34c-9995-4395-a229-055190e1d29e&swVersion=4&extensionId=eamodio.gitlens&platform=electron&vscode-resource-base-authority=vscode-resource.vscode-cdn.net&parentOrigin=vscode-file%3A%2F%2Fvscode-app&purpose=webviewView#)`

Reply:
@adisos I asked @kyorav when verifying the new outputs the relevant node here is not `iksNode` but an `vpe`, and this is indeed the `SG` associated with it, as explicitly mentioned in `endpoint_gateways`